src/autogluon/cloud/predictor/cloud_predictor.py

```diff
@@ -650,6 +650,7 @@ def predict_proba(
         return self.backend.predict_proba(
             test_data=test_data,
             test_data_image_column=test_data_image_column,
+            include_predict=include_predict,
```

Comment:
Just curious, where did this come from?

Reply:
I just noticed this variable is not being passed in after the backend update so decided to add it here